vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -316,6 +316,11 @@
       "source": "parts/linux/cloud-init/artifacts/sshd_config_1804_fips",
       "destination": "/home/packer/sshd_config_1804_fips"
     },
+    {
+      "type": "file",
+      "source": "parts/linux/cloud-init/artifacts/sshd_config_2204_fips",
+      "destination": "/home/packer/sshd_config_2204_fips"
+    },
```

Comment:
Added here as well, because the 1804 FIPS config was added too

Reply:
actually this new file is not required by the arm64 vhd build. And sshd_config_1804_fips can be removed too.

Reply:
I'll remove them both!